### PR TITLE
docs: add sway standards

### PR DESCRIPTION
**Pull request by sarahschwartz**
adds the sway standards book, and fixes other minor issues like preparing the sway-libs repo to fetch the current version once there is a new release

### CODE REVIEW

File: docs/contributing/versions.mdx [hunk 1]
```diff
@@ -8,10 +8,9 @@ parent:
 
 # Versions
 
-There are three version sets of the docs available in the `docs-hub`: `testnet`(the default version), `beta-5`, and `nightly`.
+There are three version sets of the docs available in the `docs-hub`: `testnet`(the default version), and `nightly`.
```
Comment (matt-user) [inline docs/contributing/versions.mdx:11]: are there two version sets now?

File: src/lib/plugins/links.ts [hunk 2]
```diff
@@ -69,18 +69,8 @@ export function handleLinks(
...
-            return `/libs/src/${p1}`;
-          })
-          .replace('/src/src/', '/src/');
-      } else if (newUrl.includes('/sway-standards/')) {
-        newUrl = newUrl.replace(
-          '/standards/src5-ownership',
-          '/standards/src/src5.sw'
-        );
+          .replace('/src/src/', '/src/')
+          .replace('sway-libs/tree/libs/nft','sway-libs/tree/v0.12.0/libs/nft');
```
Comment (matt-user) [inline src/lib/plugins/links.ts:73]: why was this added?

File: src/lib/plugins/links.ts [hunk 3]
```diff
@@ -312,6 +302,70 @@ function handleNewURLs(
 
 function replaceInternalLinks(href: string, base: string) {
   let newHref = href;
+  if(newHref.startsWith('https://github.com/FuelLabs/sway-standards/') && newHref.endsWith('.md') && !newHref.includes('README.md')){
+    const split = newHref.split('/');
+    let docName = split[split.length -1].replace(".md", "");
+    switch (docName) {
+      case 'src-2':
+        docName = 'src-2-inline-documentation'
```
Comment (matt-user) [inline src/lib/plugins/links.ts:310]: why do we need to change the doc name?

File: src/lib/versions.ts [hunk 4]
```diff
@@ -62,18 +62,32 @@ function getForcVersion(docsDir: string) {
...
-
-//   return {
-//     name: 'forc',
-//     category: 'Forc',
-//     version,
-//     url: `https://github.com/FuelLabs/sway/tree/v${version}`,
-//   };
-// }
+function getSwayLibsVersion(docsDir: string) {
+  // TODO: uncomment this once sway-libs 0.22.0 is released
```
Comment (matt-user) [inline src/lib/versions.ts:66]: what's important about 0.22.0 release?

File: tests/utils/fixtures/index.ts [hunk 5]
```diff
@@ -6,14 +6,21 @@ import { test as base, chromium } from '@playwright/test';
...
+
 export const test = base.extend<{
   context: BrowserContext;
   extensionId: string;
 }>({
   // biome-ignore lint/correctness/noEmptyPattern:
   context: async ({}, use) => {
     // download fuel wallet
-    const fuelPathExtension = await downloadFuel('0.16.6');
+    console.log("DOWNLOADING WALLET VERSION", versions.default.wallet)
```
Comment (matt-user) [inline tests/utils/fixtures/index.ts:22]: Do we want this log?

**Review by matt-user — COMMENTED** [review]
Overall looks good.  I just have some questions about the changes

File: src/lib/plugins/links.ts (hunk 2 above)
Comment (sarahschwartz) [reply src/lib/plugins/links.ts:73]: the link was broken

File: src/lib/plugins/links.ts (hunk 3 above)
Comment (sarahschwartz) [reply src/lib/plugins/links.ts:310]: I just changed the doc names in the source repo, which broke the old links. This patches the links until they get fixed in the other source repos

File: src/lib/versions.ts (hunk 4 above)
Comment (sarahschwartz) [reply src/lib/versions.ts:66]: the PR to release that versions also adds a Cargo.toml file that has the version of the repo. Right now the version isn't in the repo.

File: tests/utils/fixtures/index.ts (hunk 5 above)
Comment (sarahschwartz) [reply tests/utils/fixtures/index.ts:22]: yes, this just runs in the playwright tests
